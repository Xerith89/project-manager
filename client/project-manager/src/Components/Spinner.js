import React from 'react'

export default function Spinner() {
    return (
        <div className="d-flex justify-content-center m-5" data-testid='spinner'>
            <div className="spinner-border" role="status">
               <h3>Loading...</h3>
            </div>
      </div>
    )
}
