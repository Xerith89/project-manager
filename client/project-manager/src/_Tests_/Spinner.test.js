import React from 'react'
import Spinner from '../Components/Spinner'
import {shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'

configure({adapter: new Adapter()});

test('Spinner component renders to the DOM', () => {
    const wrapper = shallow(<Spinner />);
    expect(wrapper.html()).toMatchSnapshot();
});