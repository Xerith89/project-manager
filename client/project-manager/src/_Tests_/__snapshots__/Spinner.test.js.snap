// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component renders to the DOM 1`] = `"<div class=\\"d-flex justify-content-center m-5\\" data-testid=\\"spinner\\"><div class=\\"spinner-border\\" role=\\"status\\"></div></div>"`;
